03-FLEX/Portfolio/sergejs-vasjkvos.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Front-End Developer</title>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Vazirmatn:wght@300;500;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style/style.css">
+</head>
+
+<body>
+  <header class="header">
+    <nav class="header-nav container">
+
+      <a class="header__logo" href="sergejs-vasjkvos.html">
+        <img width="70" height="50" src="img/logo.svg" alt="John Doe logo, home page button">
```

Comment:
Just a tip - to optimize image load you can add lazyloading to them. https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading